src/services/service.js

```diff
@@ -125,15 +125,20 @@ module.exports = class Service {
 
     return fetch(uri, opts)
       .then((res) => {
-        return res.json().then((data) => {
-          if (res.status != 200 && res.status != 201) {
-            console.error(data);
-          }
-          return {
-            statusCode: res.status,
-            ...data,
-          };
-        });
+        if (res.status < 500) {
+          return res.json().then((data) => {
+            if (res.status != 200 && res.status != 201) {
+              console.error(data);
+            }
+            return {
+              statusCode: res.status,
+              ...data,
+            };
+          });
+        } else {
+          // 500 responses do not return a json structure
+          return res;
```

Comment:
If the service the SDK is wrapping changes the shape of it's 5XX response, will this change cause the SDK's response to also change?